server/api/categoryRouter.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Router for category actions
+ */
+const Category = require('../db/models/category');
+const express = require('express');
+const Item = require('../db/models/item');
+const router = express();
+
+/**
+ * Create a new category
+ * 
+ * req.body: {title: String}
+ * 
+ * response: {success: boolean, message: String}
+ */
+router.post('/createCategory', (req, res, next) => {
+  const newCategory = req.body;
+  let response = {};
+  Category.create(newCategory, (err, data) => {
+    if (err) {
+      response.successs = false;
+      response.message = 'failed to add category';
+      next(err);
+    } else {
+      response.success = true;
+      response.message = 'created new category';
+    }
+    res.json(response);
+  });
+});
```

Comment:
Should this have some sort of error catching around it?

Reply:
discussed blow